### PR TITLE
[yioremote] BugFix sendIRCode action

### PR DESCRIPTION
**Pull request by miloit**
## Expected Behavior
if the action sendIRCode is used in a rule it should send out an IR code

## Current Behavior
action sendIRCode is not sending out any IR code

## Possible Solution
Correct implementation of sendIR action routine
In addition:
Updating IR receiver switch handler so it's running now

Solved issue during triggering channel while it was disposed

### CODE REVIEW

File: bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/YIOremoteDockHandler.java [hunk 1]
```diff
@@ -196,12 +199,14 @@ private boolean decodeReceivedMessage(JsonObject message) {
                     success = false;
                 }
             } else if (message.get("command").toString().equalsIgnoreCase("\"ir_receive\"")) {
+                // sendMessage(YioRemoteMessages.IR_RECEIVER_OFF, "");
```
Comment (fwolter) [inline bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/YIOremoteDockHandler.java:202]: Can this be removed?

**Review by fwolter — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/YIOremoteDockHandler.java (hunk 1 above)
Comment (miloit) [reply bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/YIOremoteDockHandler.java:202]: yes

Comment (miloit) [reply bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/YIOremoteDockHandler.java:202]: done

**Review by fwolter — APPROVED** [review]
LGTM

**Pull request merged**